src/main/java/somethingrandom/dataaccess/local/FileUserDataAccessObject.java

```diff
@@ -0,0 +1,131 @@
+package somethingrandom.dataaccess.local;
+
+import org.json.JSONObject;
+import org.json.JSONWriter;
+import somethingrandom.entity.*;
+import somethingrandom.usecase.DataAccessException;
+
+import java.io.*;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.*;
+
+public class FileUserDataAccessObject {
+    private final File dataFile;
+    private final Map<UUID, Map<String, String>> items;
+    private final ItemFactory factory;
+
+    public FileUserDataAccessObject(String dataFilePath, ItemFactory factory) throws DataAccessException {
+        dataFile = new File(dataFilePath);
+        items = new HashMap<>();
+        this.factory = factory;
+        if (dataFile.length() == 0) {
+            save();
+        } else {
+            load();
+        }
```

Comment:
In a way, I'd prefer for this to be 'lazy': load and save only when we actually want the data. I also suspect we could always load, then always save, but this should work.